services/storage/src/simcore_service_storage/api/rpc/_async_jobs.py

```diff
@@ -36,13 +37,17 @@ async def get_status(
 ) -> AsyncJobStatus:
     assert app  # nosec
     assert job_id_data  # nosec
-    progress_report = ProgressReport(actual_value=0.5, total=1.0, attempt=1)
+
+    task_status: TaskStatus = await get_celery_client(app).get_task_status(
+        task_context=job_id_data.model_dump(),
```

Comment:
I'm going to comment here since elsewre I can't. the `AsyncJobNameData` looks strage to me. It's used in a generic interface but has fixed parameters. It should be a `dict[str, Any]` in my opinion. Some tasks may require different arugments.